constants.py

```diff
@@ -29,7 +29,7 @@
 )
 
 # Context Window and Max New Tokens
-CONTEXT_WINDOW_SIZE = 8096
+CONTEXT_WINDOW_SIZE = 2048
```

Comment:
@siddhivelankar23  Is it necessary to adjust the context window to a smaller size, or did it change by accident?

Reply:
Sounds good. It should be restored back to 8096 tokens since that's not intended changes by this PR. 

I raised a pull request to restore the config: https://github.com/PromtEngineer/localGPT/pull/832

Pending on @PromtEngineer to review.